milkyway/assetlist.json

```diff
@@ -286,6 +286,73 @@
           }
         }
       ]
+    },
+    {
+      "description": "Drop staked TIA",
+      "denom_units": [
+        {
+          "denom": "ibc/4795C86F72EC2CB54E885A4A5B401BED9C333E008311ED9C4DE01BF546B44A21",
+          "exponent": 0,
+          "aliases": [
+            "udtia"
+          ]
+        },
+        {
+          "denom": "dTIA",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/4795C86F72EC2CB54E885A4A5B401BED9C333E008311ED9C4DE01BF546B44A21",
+      "name": "Stride Staked TIA",
+      "display": "stTIA",
```

Comment:
this unit isn't defined among the denom_units

Reply:
Fixed it, thank you!